README.md

```diff
@@ -0,0 +1,201 @@
+<h1 align="center">go-slack-chat-gpt</h1>
+<p align="center">
+    <a href="https://github.com/drkennetz/go-slack-chat-gpt3/graphs/contributors" alt="Contributors">
+        <img src="https://img.shields.io/github/contributors/drkennetz/go-slack-chat-gpt3.svg" /></a>
+    <a href="https://github.com/drkennetz/go-slack-chat-gpt3/pulse" alt="Activity">
+        <img src="https://img.shields.io/github/commit-activity/m/drkennetz/go-slack-chat-gpt3" /></a>
+    <a href="https://github.com/drkennetz/go-slack-chat-gpt3">
+        <img alt="GitHub Workflow Status" src="https://img.shields.io/github/actions/workflow/status/drkennetz/go-slack-chat-gpt3/ci.yml">
+    </a>
+<!--    <a href="https://github.com/drkennetz/go-slack-chat-gpt3">
+        <img src="https://img.shields.io/github/workflow/status/drkennetz/go-slack-chat-gpt3/Service%20Testing/main" alt="Service Testing Status">
+    </a> -->

```

Comment:
@drkennetz can you please remove these commented line. They are created by me by mistake while I am adding badges to README.md file.

Reply:
@hemantsoni42 just a note, if you have finished the review - click "request changes" so I know that I can begin work.